tutorials/shaders/shader_reference/canvas_item_shader.rst

```diff
@@ -117,7 +117,7 @@ is usually:
 |                                | For a Sprite2D with a texture of size 64x32px,     |
 |                                | **TEXTURE_PIXEL_SIZE** = :code:`vec2(1/64, 1/32)`  |
 +--------------------------------+----------------------------------------------------+
-| inout vec2 **VERTEX**          | Vertex, in image space.                            |
+| inout vec2 **VERTEX**          | Vertex, in canvas space.                           |
```

Comment:
This should say "local" space. Canvas space is relative to the screen. While the coordinates of the VERTEX are relative to the Node2D and use a pixel scale. The rest of this doc refers to that space as "local" space

Reply:
Thank you, I will update this.

Reply:
Done.